src/index.js

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Messenger from './pages/Messenger';
-import registerServiceWorker from './registerServiceWorker';
+import Commit from 'pages/Commit';
+
+import registerServiceWorker from 'registerServiceWorker';
 
 import {
-    BrowserRouter as Router,
-    Route,
+  BrowserRouter as Router,
+  Route
 } from 'react-router-dom';
 
+
+import './index.css';
```

Comment:
Should we stick with styled components as a standard for this project?

Reply:
We should use both. In this case the global css is used to style body and generic tags like h1, h2 etc... Styled component is for per component styling.

Reply:
k